timemachine/cpp/src/local_md_potentials.cu

```diff
@@ -220,6 +239,26 @@ void LocalMDPotentials::_setup_free_idxs_given_reference_idx(
     // Free particles should be in the row idxs
     set_nonbonded_ixn_potential_idxs(
         ixn_group_->potential, num_col_idxs, num_row_idxs, d_col_idxs_.data, d_row_idxs_.data, stream);
+
+    if (!freeze_reference_) {
+        // If there are no frozen, don't attach any
```

Comment:
nit: clarify comment (should contain "indices"?)

Reply:
Good catch, fixed up the comment [e0a0819](https://github.com/proteneer/timemachine/pull/1014/commits/e0a0819e25379d3549ca2bc4dfd40b2c71bd127e)